sphinxgallery/gen_rst.py

```diff
@@ -987,6 +1040,9 @@ def embed_code_links(app, exception):
         'numpy': 'http://docs.scipy.org/doc/numpy-1.6.0',
         'scipy': 'http://docs.scipy.org/doc/scipy-0.11.0/reference',
     }
+    if has_mayavi:
```

Comment:
The new version of sphinx-gallery handles this urls as user input. Shall this considered as default documented package?